app/com/arpnetworking/metrics/portal/alerts/impl/DatabaseAlertRepository.java

```diff
@@ -239,7 +239,7 @@ private Alert convertFromEbeanAlert(final models.ebean.Alert ebeanAlert) {
                 .setMetric(ebeanAlert.getMetric())
                 .setName(ebeanAlert.getName())
                 .setOperator(ebeanAlert.getOperator())
-                .setPeriod(Period.seconds(ebeanAlert.getPeriod()))
+                .setPeriod(Period.seconds(ebeanAlert.getPeriod()).normalizedStandard())
```

Comment:
Thanks.  Turns out that test failure was the same thing happening in the test code.  ) :